training_pipeline/pipeline/components/HFPusher/model_card.py

```diff
@@ -0,0 +1,28 @@
+from huggingface_hub import ModelCard, ModelCardData
+
+
+def create_card(card_data, template_path, model_metadata, 
+                placeholders, **template_kwargs):
+    """Creates model card.
+
+    Args:
+        model_metadata (dict): Dict of card metadata.
+        see here for what you can pass to the metadata section:
+        https://github.com/huggingface/huggingface_hub/blob/main/src/huggingface_hub/templates/modelcard_template.md
+
+    Returns:
+        Model card (ModelCard): instantiated and filled ModelCard.
+    """
+    if model_metadata is None:
+        model_metadata = {}
+    model_metadata["library_name"] = "vertex"
```

Comment:
If one runs the pipeline on a different orchestrator then `vertex` won't make sense. Also, it's better to call the key `orchestrator` or `runner` to denote that it's the platform on which things are getting executed.